packages/core/src/external/carequality/dq/create-metadata-xml.ts

```diff
@@ -173,7 +174,9 @@ export function createExtrinsicObjectXml({
     </ExternalIdentifier>
     
     <!-- (IHE) REQUIRED - DocumentEntry.uniqueId - Globally unique identifier assigned to the document by its creator -->
-    <ExternalIdentifier id="${uuidv7()}" identificationScheme="urn:uuid:2e82c1f6-a085-4c72-9da3-8640a32e42ab" objectType="urn:oasis:names:tc:ebxml-regrep:ObjectType:RegistryObject:ExternalIdentifier" registryObject="${documentUUID}" value="${documentUniqueId}">
+    <ExternalIdentifier id="${uuidv7()}" identificationScheme="urn:uuid:2e82c1f6-a085-4c72-9da3-8640a32e42ab" objectType="urn:oasis:names:tc:ebxml-regrep:ObjectType:RegistryObject:ExternalIdentifier" registryObject="${documentUUID}" value="${createDocumentUniqueId(
+    documentUniqueId
+  )}">
```

Comment:
Too convoluted, let's create consts w/ the valued to be used when building the XML please